cfme/tests/automate/test_common_methods.py

```diff
@@ -105,14 +108,19 @@ def test_vm_retire_extend(request, testing_vm, soft_assert, retire_extend_button
         soft_assert(set_date == retirement_date.strftime(pick(VM.RETIRE_DATE_FMT)),
                     "The retirement date '{}' did not match expected date '{}'"
                     .format(set_date, retirement_date.strftime(pick(VM.RETIRE_DATE_FMT))))
-    # current_retirement_date = testing_vm.retirement_date
 
-    # Now run the extend stuff
+    # Create the vm_retire_extend button and click on it
     retire_extend_button()
 
-    # dajo - 20140920 - this fails because its not turning the calendar to the next month?
-    # wait_for(
-    #     lambda: testing_vm.retirement_date >= current_retirement_date + timedelta(days=14),
-    #     num_sec=60,
-    #     message="extend the retirement date by 14 days"
-    # )
+    # CFME automate vm_retire_extend method defaults to extending the date by 14 days
+    extend_duration_days = 14
```

Comment:
I'd much rather pull this from appliance configuration or something like that, since it could easily be changed from the default. This is fine for now though, later enhancements to the test (set button to non-default interval and make sure the right interval is applied)